superset-frontend/src/dashboard/components/CodeModal.tsx

```diff
@@ -17,24 +17,30 @@
  * under the License.
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import { t } from '@superset-ui/core';
 
-import ModalTrigger from '../../components/ModalTrigger';
+import ModalTrigger from 'src/components/ModalTrigger';
 
-const propTypes = {
-  triggerNode: PropTypes.node.isRequired,
-  code: PropTypes.string,
-  codeCallback: PropTypes.func,
+type CodeModalProps = {
+  triggerNode: JSX.Element;
+  code?: string;
```

Comment:
as mentioned here: i think this can be required now: https://github.com/apache/incubator-superset/pull/11971/files#r540334777